UnityProject/Assets/Scripts/Variable Viewer/BookViewer/ElementDisplay/ElementTypes/Vectors/GUI_P_Vectors.cs

```diff
@@ -0,0 +1,252 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.UI;
+using DatabaseAPI;
+
+public class GUI_P_Vectors : PageElement
```

Comment:
This should probably have documentation

Reply:
(added Some)  is there any more parts that need documentation?

Reply:
Added some is there any parts need more documentation?

Reply:
Um, Sure! How about `UpdateClientValue.cs`?

I'm used to StyleCop nagging me, so you probably don't need to document every method. But more docs are always good! :)